server/src/main/java/org/apache/druid/server/log/FileRequestLoggerProvider.java

```diff
@@ -58,7 +58,8 @@
   @Override
   public RequestLogger get()
   {
-    FileRequestLogger logger = new FileRequestLogger(jsonMapper, factory.create(1, "RequestLogger-%s"), dir, filePattern);
+    FileRequestLogger logger = new FileRequestLogger(jsonMapper, factory.create(1, "RequestLogger-%s"), dir,
+        filePattern);
```

Comment:
Please chop down all arguments (each on its own line) if they don't fit a line.

Reply:
got it. I've got to stop forgetting about that one, thanks for the comment